readme.md

```diff
@@ -24,13 +24,13 @@ You've written the following HTML. When you look at it in your browser, it's jus
 </html>
 ```
 
-> Your answer...
+> The title tag in the head does not have a </title> close tag so it considers everything to be in the title which wont show on the page.
 
 ## Question 2
 
 What's the purpose of the `alt` attribute on image tags?
 
-> Your answer...
+> It is a description of the image that usually shows when you hover over the image. They are so blind people have an idea what the image is too. Page readers can read them.
```

Comment:
They usually don't show when you hover, actually. Maybe you're thinking of the `title` attribute?

Also, the `alt` attribute is what is shown whenever the image can't be viewed for any reason -- such as when the link is broken.